modules/samples/artifacts/CSVMappingWithFile/CSVCustomMapping.siddhi

```diff
@@ -0,0 +1,65 @@
+@App:name("CSVCustomMapping")
+@App:description('Use inmemory transport to custom CSV mapping and the custom CSV mapping and view the output on the console.')
+
+@App:description('Publish and receive data events processed within Siddhi to files in CSV custom format.')
+
+/*
+
+Purpose:
+	This application demonstrates how to configure WSO2 Stream Processor to Publish and receive data events processed within Siddhi to files in CSV custom format.
+    
+Prerequisites:
+    * Edit the uri '{WSO2SPHome}/samples/artifacts/CSVMappingWithFile/new/{{name}}.txt' by replacing {WSO2SPHome} with the absolute path of your WSO2SP home directory. You can also change the path after {WSO2SPHome} (/samples/artifacts/CSVMappingWithFile/new/) if you want to publish your event file to a different location.
+
+	* Save this sample. If there is no syntax error, the following messages would be shown on the console:
+	     * - CSVCustomMapping.siddhi successfully deployed.
+    
+Executing the Sample:
+	1) Start the Siddhi application by clicking on 'Run'.
+	2) If the Siddhi application starts successfully, the following messages are shown on the console:
+            * CSVCustomMapping.siddhi - Started Successfully!
+
+Testing the Sample:
+    You can publish data events to the file through the event simulator:
+        1) Click on 'Event Simulator' (double arrows on left tab)
+        2) Click 'Single Simulation' (this will be already selected)
+        3) Select CSVCustomMapping as 'Siddhi App Name'
+        4) Select InMemoryCustomSweetProductionInputData as 'StreamName'
+        5) Click on the start button (Arrow symbol) next to the newly created simulator
+        6) Provide attribute values
```

Comment:
Do we need to provide attributes and send ???

Reply:
We defined  stream with attributes so we have to need to provide  that attributes's values, Otherwise simulator will give the message such as "This field is required".

Reply:
The question is why we need to use simulator here, you add the file to the file path given in source and you get the new file in the file path provided in the sink. so why we need event simulator?